maltoolbox/language/languagegraph.py

```diff
@@ -294,8 +388,25 @@ def to_dict(self) -> dict:
                 else:
                     node_dict['parents'][parent].append(None)
 
+        if hasattr(self, 'requirements'):
+            node_dict['requirements'] = []
+            for requirement in self.requirements:
+                node_dict['requirements'].append(requirement.to_dict())
+
```

Comment:
I think the attribute should be names `requires` to keep the convention that other attributes established (e.g. `inherits`, `overrides`) as well as conform to what is in other parts of the code like what gets returned from the compiler that calls this `requires`.